src/Modules/ModParser.lua

```diff
@@ -3301,7 +3302,7 @@ local flagTypes = {
 	["blinded"] = "Condition:Blinded",
 	["no life regeneration"] = "NoLifeRegen",
 	["hexproof"] = { name = "CurseEffectOnSelf", value = -100, type = "MORE" },
-	["hindered, with 25% reduced movement speed"] = "Condition:Hindered",
+	["hindered, with (%d+)%% reduced movement speed"] = "Condition:Hindered",
```

Comment:
Unfortunately, this table doesn't work the same as `specialMods`, as it's an exact string lookup.  You can change the parameter to make it work with patterns (not sure why it wasn't implemented that way at this point, though):
https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/063a993a87f34ec89bb3ae4ad9195e49529dea26/src/Modules/ModParser.lua#L3788